### PR TITLE
Send user

### PR DESCRIPTION
**Pull request by wcraig93**
[#165100183]-queues still use factories but now the urls are transferred in from dependencies.

### CODE REVIEW

File: src/Housing.Selection.Service/appsettings.Development.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "Settings": {
-    "Queues": false,
+    "Queues": true,
```
Comment (stormrubio3000) [inline src/Housing.Selection.Service/appsettings.Development.json]: this should be false when pushing to delopyment

File: src/Housing.Selection.Service/appsettings.json [hunk 2]
```diff
@@ -4,8 +4,14 @@
     "DbInDockerMachineContainer": "Server=192.168.99.100;Database=HousingSelectionDb;User Id=postgres;"
   },
   "Settings": {
+    "Queues": "true",
```
Comment (stormrubio3000) [inline src/Housing.Selection.Service/appsettings.json]: here as well

File: src/Housing.Selection.Service/appsettings.Development.json (hunk 1 above)
Comment (wcraig93) [reply src/Housing.Selection.Service/appsettings.Development.json]: ? if I make it false then it doesnt use the queues

Comment (wcraig93) [reply src/Housing.Selection.Service/appsettings.Development.json]: but ok

File: src/Housing.Selection.Testing/Context/HttpRequestsTests.cs [hunk 3]
```diff
@@ -208,5 +274,219 @@ public async Task GetAllUsers_StatusCodeFail_ReturnsNull()
...
+            var mockApiPath = new Mock<IApiPathBuilder>();
+            mockApiPath.Setup(x => x.GetUserServicePath()).Returns("string");
+
+            var src = new ServiceUserCalls(mockHttpWrapper.Object, mockApiPath.Object);
+
+            await Assert.ThrowsAsync<InvalidOperationException>(() => src.UpdateUserAsync(user));
+        }
     }
-}
+}
```
Comment (stormrubio3000) [inline src/Housing.Selection.Testing/Context/HttpRequestsTests.cs]: need an extra line for lenting

**Pull request merged**